start.sh

```diff
@@ -1,10 +1,48 @@
 #!/bin/bash
 
-# start peacock
-cd Peacock
+# Get the directory of the script
+SCRIPT_DIR=$(dirname "$(readlink -f "$0")")
+pushd $SCRIPT_DIR
 
-PORT=3000 ../node/bin/node ./chunk0.js &
+# Grab Peacock if needed
+if [ ! -f "./Peacock/chunk0.js" ]; then
+    LATEST_RELEASE=$(curl -L -s -H 'Accept: application/json' https://github.com/thepeacockproject/peacock/releases/latest | sed -e 's/.*"tag_name":"\([^"]*\)".*/\1/')
 
-# start game command
-../run
+    wget -O peacock.zip "https://github.com/thepeacockproject/Peacock/releases/download/${LATEST_RELEASE}/Peacock-${LATEST_RELEASE}.zip"
```

Comment:
Using the `-lite` build will help saving space by not downloading the node.exe built-in the normal build
`"https://github.com/thepeacockproject/Peacock/releases/download/${LATEST_RELEASE}/Peacock-${LATEST_RELEASE}-lite.zip"`

Reply:
I skimmed through the megathread on the discord and saw the -lite build be not recommended, I wasn't aware of the differences but if that's it then agreed

Reply:
@RDIL should be able to confirm it but I'm almost certain it's the only diff between both

Reply:
Yeah that's the only difference, but it's being renamed to `linux` instead of `lite` in the next version so for now this can remain the full build so that it doesn't get broken when that change gets into the latest stable release.